packages/babel-types/src/retrievers/getBindingIdentifiers.ts

```diff
@@ -117,10 +117,6 @@ getBindingIdentifiers.keys = {
   ImportDefaultSpecifier: ["local"],
   ImportDeclaration: ["specifiers"],
 
-  ExportSpecifier: ["exported"],
```

Comment:
When calling `path.remove()`, `@babel/traverse` will unregister from the scope all the bindings introduced by the removed path.

Consider this case:
```js
import { x, y } from "x";
export { type x, y }
```

when calling `path.remove()` on `type x`, we don't want to unregister `x` from the scope since it's real binding is in the import declaration.

I initially just hacked around `path.remove()` in the TS plugin, but this solution is better because export specifiers don't actually introduce bindings in the current scope.